docs/standard/base-types/custom-numeric-format-strings.md

```diff
@@ -191,23 +195,55 @@ You can create a custom numeric format string, which consists of one or more cus
  The following example uses the ";" format specifier to format positive, negative, and zero numbers differently.  
   
  [!code-cpp[Formatting.Numeric.Custom#8](../../../samples/snippets/cpp/VS_Snippets_CLR/formatting.numeric.custom/cpp/custom.cpp#8)]
- [!code-csharp[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
+ [!code-csharp-interactive[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
  [!code-vb[Formatting.Numeric.Custom#8](../../../samples/snippets/visualbasic/VS_Snippets_CLR/formatting.numeric.custom/vb/Custom.vb#8)]  
   
  [Back to table](#table)  
-  
+
+## Character literals  
+ 
+The following characters in a custom numeric format string are reserved and are always interpreted as formatting characters:  
```

Comment:
All those characters are described at the table at the start of the topic where they are called *format specifiers*
Should we use the introduced term?

Reply:
Yes, we should, @pkulikov.